### ISSUE TITLE
Add new feedback items to the end

### ISSUE THREAD

**Issue by earthling2** [microsoft/vsts-extension-retrospectives#507]
**Is your feature request related to a problem? Please describe.**
When we are in retrospective and reviewing items, if someone enters a new feedback item it is added to the beginning of the feedback list and changes the order of the item being reviewed (throwing off the person who is presenting). The person reviewing has to arrow back in order to get to the newly added items.

**Describe the solution you'd like**
Add new feedback items to the end of each column. Do not have the review screen change the current item being reviews, instead you can add it to the end of the items and they will come to it using the forward arrows. Nice to have: If someone already reviewed that column, show a badge with a number of the new items. Allow the reviewer to click on the badge and see only the newly added (non-reviewed) items.

**Describe alternatives you've considered**
Making sure that all feedback is entered before starting review. However, there are times when you think of something during the review so this doesn't follow real-life use for us.

**Additional context**
